charts/opensearch/templates/statefulset.yaml

```diff
@@ -211,8 +211,20 @@ spec:
       {{- if .Values.hostAliases }}
       hostAliases: {{ toYaml .Values.hostAliases | nindent 8 }}
       {{- end }}
-      {{- if or (.Values.extraInitContainers) (.Values.keystore) }}
       initContainers:
+{{- if .Values.persistence.enabled }}
+      - name: fsgroup-volume
+        image: busybox
+        command: ['sh', '-c']
```

Comment:
Won't this break? `initContainers` will be written but if `persistence.enabled=`false`, `extraInitContainers=[]` AND `keystore.enabled=false` are missing. It's empty in the YAML.

Reply:
indeed @DandyDeveloper !  it should be init if any of them applies, now should be fixed